### DISCUSSION TITLE
How do I cancel/stop a transaction/update?

### DISCUSSION THREAD

**Discussion by einarpersson** [ueberdosis/tiptap#5822] [General Questions]
Hi!

I am using tiptap with https://github.com/sibiraj-s/svelte-tiptap but I think this question is a general tiptap-question

How can in an event stop/cancel a current transaction/update? I am able to capture events onTransaction and onUpdate and I see many props/methods but nothing that says "cancel" or "stop" or something of that sort.

I have also looked in the docs but I only see the handlers, but nothing relating to canceling / stopping 
https://tiptap.dev/docs/editor/api/events

Grateful for any help!

**Comment by nperez0111** [comment]
Prosemirror plugins are able to `filterTransaction`s, https://prosemirror.net/docs/ref/#state.PluginSpec.filterTransaction which would allow you to stop a transaction from being applied to the view

So you can create a prosemirror plugin in Tiptap to do that: https://tiptap.dev/docs/editor/extensions/custom-extensions/extend-existing#access-the-prosemirror-api
